task2/spichek.c

```diff
@@ -0,0 +1,135 @@
+#include <stdio.h>
+#include <locale.h>
+#include <time.h>
+
+f2() {   //2 �����
```

Comment:
Function should have type, if function doesn't return anything, it should return void
`void f2()`